OWNERS

```diff
@@ -5,6 +5,21 @@ filters:
     reviewers:
       - code-reviewers
     approvers:
-      - approvers
+      - sig-architecture-approvers
+      - sme-live-migration
+      - sme-api-n-controllers
+      - sme-scheduling-n-node-plumbing
+      - sme-scale-n-observability
+      - sme-ci-n-testing
+      - sme-buildsystem
+      - sme-release
     emeritus_approvers:
       - emeritus_approvers
+
+  "hack/.*|tests/.*|.*_test.go":
```

Comment:
How about adding an entry for `automation` folder?
```suggestion
  "automation/.*":
    approvers:
      - sme-buildsystem
  "hack/.*|tests/.*|.*_test.go":
```

Reply:
God idea!